src/test/java/com/marklogic/kafka/connect/source/AbstractIntegrationSourceTest.java

```diff
@@ -108,6 +116,21 @@ private void assertTopicAndSingleValue(List<SourceRecord> newSourceRecords, Stri
             "List of SourceRecords does not contain a record with the expected value");
     }
 
+    private void assertRecordKeys(List<SourceRecord> newSourceRecords, String keyStrategy) {
+        if(keyStrategy.equalsIgnoreCase("uuid")) {
+            newSourceRecords.forEach(sourceRecord -> Assertions.assertEquals(UUID.fromString(sourceRecord.key().toString()).toString(), sourceRecord.key().toString()));
```

Comment:
Use `assertEquals` instead of `Assertions.assertEquals` - see https://wiki.marklogic.com/display/PM/Writing+Tests+for+Developer+Experience for a tip on how to do this easily in Intellij.